src/modules/dashboard/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -1,26 +1,66 @@
 import React from 'react';
+import { defineMessages } from 'react-intl';
 
-import { Domain } from '~data/index';
+import { SpinnerLoader } from '~core/Preloaders';
+import { useColonySuggestionsQuery } from '~data/index';
 import { Address } from '~types/index';
+import Heading from '~core/Heading';
+import { getMainClasses } from '~utils/css';
+
+import styles from './SuggestionsList.css';
+
+const MSG = defineMessages({
+  emptyStateTitle: {
+    id: 'Dashboard.SuggestionsList.emptyStateTitle',
+    defaultMessage: 'There are no suggestions here.',
+  },
+  emptyStateSubTitle: {
+    id: 'Dashboard.SuggestionsList.emptyStateSubTitle',
+    defaultMessage:
+      'Create a new suggestion, or switch domains to change the filter.',
+  },
+});
 
 interface Props {
   colonyAddress: Address;
-  domainId: Domain['ethDomainId'];
 }
 
 const displayName = 'Dashboard.SuggestionsList';
 
-const SuggestionsList = ({ colonyAddress, domainId }: Props) => {
-  // @todo use suggestions query here
-  const suggestions = ['okay', 'alright'];
+const SuggestionsList = ({ colonyAddress }: Props) => {
+  const { data, loading } = useColonySuggestionsQuery({
+    variables: { colonyAddress },
+  });
+  if (loading) {
+    return <SpinnerLoader size="medium" />;
+  }
+
+  const suggestions = (data && data.colony && data.colony.suggestions) || [];
```

Comment:
if `data` exists, `colony` will exist as well and `colony.suggestions`, too. It seems we can simplify a bit:

`data && data.colony.suggestions` (suggestions is always defined, fallback is an empty array anyways).

Reply:
Nice catch